packages/fx-core/src/core/SolutionPluginContainer.ts

```diff
@@ -40,6 +42,14 @@ export function getAllSolutionPlugins(): Solution[] {
   return plugins;
 }
 
+export function getSolutionPluginByCap(cap: string[]): v2.SolutionPlugin | undefined {
```

Comment:
Since solution select question is removed, currently the map between capability and solution is hard coded here. Later we can extend the function of question model and set selected solution when a certain tree node is traversed.